identity/src/main/java/com/stripe/android/identity/ui/ConsentScreen.kt

```diff
@@ -75,58 +74,56 @@ internal fun ConsentScreen(
     val context = LocalContext.current
     val coroutineScope = rememberCoroutineScope()
 
-    MdcTheme {
-        CheckVerificationPageAndCompose(
-            verificationPageResource = verificationPageState,
-            onError = {
-                identityViewModel.errorCause.postValue(it)
-                navController.navigateToErrorScreenWithDefaultValues(context)
+    CheckVerificationPageAndCompose(
+        verificationPageResource = verificationPageState,
+        onError = {
+            identityViewModel.errorCause.postValue(it)
+            navController.navigateToErrorScreenWithDefaultValues(context)
+        }
+    ) {
+        val verificationPage = remember { it }
+        if (verificationPage.isUnsupportedClient()) {
+            LaunchedEffect(Unit) {
+                fallbackUrlLauncher.launchFallbackUrl(verificationPage.fallbackUrl)
             }
-        ) {
-            val verificationPage = remember { it }
-            if (verificationPage.isUnsupportedClient()) {
-                LaunchedEffect(Unit) {
-                    fallbackUrlLauncher.launchFallbackUrl(verificationPage.fallbackUrl)
+        } else {
+            LaunchedEffect(Unit) {
+                identityViewModel.updateAnalyticsState { oldState ->
+                    oldState.copy(
+                        requireSelfie = verificationPage.requireSelfie()
+                    )
                 }
-            } else {
-                LaunchedEffect(Unit) {
-                    identityViewModel.updateAnalyticsState { oldState ->
-                        oldState.copy(
-                            requireSelfie = verificationPage.requireSelfie()
+            }
+            ScreenTransitionLaunchedEffect(
+                identityViewModel = identityViewModel,
+                screenName = SCREEN_NAME_CONSENT
+            )
+            SuccessUI(
+                identityViewModel.verificationArgs.brandLogo,
+                verificationPage,
+                onConsentAgreed = {
+                    coroutineScope.launch {
```

Comment:
hmm I think it's OK because `postVerificationPageDataAndMaybeNavigate` post an API and triggers navigation(which would make `ConsentScreen` leaving the composition)